packages/adapter-cloudflare-workers/index.js

```diff
@@ -69,7 +69,11 @@ export default function ({ config = 'wrangler.toml' } = {}) {
 				entryPoints: [`${tmp}/entry.js`],
 				outfile: main,
 				bundle: true,
-				external: ['__STATIC_CONTENT_MANIFEST', 'cloudflare:*'],
+				external: [
+					'__STATIC_CONTENT_MANIFEST',
+					'cloudflare:*',
+					...(options.nodeCompat ? ['node:*'] : [])
```

Comment:
Whitelisting `node:*` may be too aggressive. I think if we do that and someone uses an API like `node:fs` then they won't find out their app is broken until they go to deploy it and it'd be better for that to occur during build. How about we switch it to just the list of APIs that Cloudflare supports with their node compatibility?

'node:assert', 'node:async_hooks', 'node:buffer', 'node:crypto', node:diagnostics_channel', node:events', node:path', node:process', node:stream', node:string_decoder', node:util'

Reply:
Yep that sounds legit. Cloudflare node_compat doesn't have modules like `fs`, ...

Reply:
Is fixed in my latest commit

Reply:
May as well I remove `AdapterOptions.nodeCompat` and always included those externals? 🤔